CMakeLists.txt

```
@@ -544,7 +544,49 @@ else()
     set(CPACK_SOURCE_GENERATOR "TGZ")
 endif(WIN32)
 set(CPACK_SOURCE_PACKAGE_FILE_NAME "${ASSEMBLY_BASE_NAME}-source")
-set(CPACK_SOURCE_IGNORE_FILES "/docs/generated/;${CMAKE_SOURCE_DIR}/build/;~$;${CPACK_SOURCE_IGNORE_FILES};${CMAKE_SOURCE_DIR}/.git/;${CMAKE_SOURCE_DIR}/.idea/;${CMAKE_SOURCE_DIR}/cmake-build-debug/")
+set(CPACK_SOURCE_IGNORE_FILES
+    "/build/"
```

I think this could be a wildcard to match more like in the .gitignore, but I'm not sure of the syntax here. I often have build directories named `build-gcc-make`, `build-gcc-ninja`, `build-clang-ninja` in addition to the CLion-generated and bootstrap-generated ones.
```suggestion
    "/.*build.*/"
```

---

Yes, good idea. I was worried about `win_build_vs.bat`, but the `/` at the end excludes (i.e. includes) that.

I have also added `build_description.cpp` to the ignore list, as that is also a generated file. e4ce729b09f1172902880adf226d4d6a697b2928